lib/reporters/tap.js

```diff
@@ -27,6 +27,15 @@ TapReporter.prototype.start = function () {
 TapReporter.prototype.test = function (test) {
 	var output;
 
+	var directive = '';
+	var passed = test.todo ? 'not ok' : 'ok';
+
+	if (test.todo) {
+		directive = '# TODO';
+	} else if (test.skipped) {
```

Comment:
This should be `test.skip`, from <https://github.com/BarryThePenguin/ava/blob/bb3c972cc4df4b3df0139b5d6f738cb18da0e6e7/lib/runner.js#L99>.